test/test_gapic_utils.py

```diff
@@ -275,60 +277,9 @@ def test_library_gapic_v2():
 
     request.proto_file.extend(file_descriptor_set.file)
 
-    reconstructed_gapic_config = gapic_utils.reconstruct_gapic_yaml(
+    gapic_config = gapic_utils.create_gapic_config_v2(
         gapic_yaml, request)
-    assert [c for c in [
```

Comment:
Where did the asserts go? I understand that there were substantial changes in the implementaiton, but do we still test (somewhere else) what was tested here?

Reply:
For generating resource names from Gapic v2 + proto annotations, we used to do it in two steps:
1) reconstruct a memory storage of a v1 GAPIC YAML;
2) create a GapicConfig object (view model) from the reconstructed v1 GAPIC YAML, and generate classes from this and the templates.

The asserts here is to make sure the reconstructed v1 GAPIC YAML is correct.

However it becomes harder and harder to continue this approach as the new multi-pattern resource name requires more information not directly stored in v1 GAPIC YAML (e.g., the actual pattern strings of a `collection_oneof` from proto annotations), so I decided to construct the viewmodel directly from v2 GAPIC YAML and annotations.

And therefore we don't need this assert any more.